src/main/resources/com/typesafe/sbt/packager/archetypes/java_server/systemloader/systemv/start-debian-template

```diff
@@ -12,6 +12,10 @@
 source /lib/init/vars.sh
 source /lib/lsb/init-functions
 
+# if configuration files exist, source it and use JAVA_OPTS to prepend their contents to $@
+[[ -f /etc/default/${{app_name}} ]] && . /etc/default/${{app_name}}
+# $JAVA_OPTS used in $RUN_CMD wrapper
+export JAVA_OPTS
```

Comment:
Sounds reasonable. So the user has to make sure that preexisting `JAVA_OPTS`
are taken into account.